.rubocop_todo.yml

```diff
@@ -365,7 +365,7 @@ Metrics/ClassLength:
 # Offense count: 76
 # Configuration parameters: IgnoredMethods.
 Metrics/CyclomaticComplexity:
-  Max: 21
+  Max: 15
 
```

Comment:
Inlined the errors for the worst offenders. Mostly triggered because I brought *the* worst offender (Which already had an inline todo) down below this threshold, and it felt silly removing the todo.